prow/plugins/config.go

```diff
@@ -274,6 +274,10 @@ type Approve struct {
 	// The default value is "https://go.k8s.io/bot-commands". The command help page is served by Deck
 	// and available under https://<deck-url>/command-help, e.g. "https://prow.k8s.io/command-help"
 	CommandHelpLink string `json:"commandHelpLink"`
+
+	// PrProcessLink is the link to the help page which explains the code review process.
+	// The default value is "https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process".
+	PrProcessLink string `json:"prProcessLink"`
```

Comment:
Can we please use camel case like for all other attributes and `omitempty`? I know I missed this in the review of `CommandHelpLink` but thats no reason to keep doing the wrong thing :)

Reply:
Absolutely! Fair point, although I'm slightly confused and assume you're referencing the json attributes that all seem to use underscores instead of Camel Case? So something like this? Sorry for the follow up question!

```suggestion
	PrProcessLink string `json:"pr_process_link,omitempty"`
```

Reply:
Yeah exactly

Reply:
Cool will do thanks!